### ISSUE TITLE
update readme

### ISSUE THREAD

**Issue by abuxton** [puppetlabs/puppet_litmus#70]
The wiki links in the readme are wrong and generate create new page attempts when used. 

correct code;

```
* [Overview](https://github.com/puppetlabs/puppet_litmus/wiki/Overview-of-Litmus) of the main functions
* [Architecture](https://github.com/puppetlabs/puppet_litmus/wiki/Architecture-of-puppet-litmus) with an explanation of what's going on under the hood
* [Step-by-step guide](https://github.com/puppetlabs/puppet_litmus/wiki/Tutorial:-use-Litmus-to-execute-acceptance-tests-with-a-sample-module-(MoTD)) of how to use Litmus with the popular and simple [MoTD Puppet module](https://forge.puppet.com/puppetlabs/motd).
* [How to guide](https://github.com/puppetlabs/puppet_litmus/wiki/Converting-a-module-to-use-Litmus) walking through how to use Litmus in a module
```

**Comment by abuxton** [comment]
https://github.com/puppetlabs/puppet_litmus/pull/71

**Comment by ekohl** [comment]
Closing since #71 was merged. For future reference, you can add `Fixes #71` to the PR message to automatically close on merge.
